components/job-orchestration/job_orchestration/scheduler/job_config.py

```diff
@@ -28,11 +28,17 @@ class FsInputConfig(BaseModel):
     timestamp_key: typing.Optional[str] = None
 
 
-class S3InputConfig(S3Config):
+class S3InputConfig(BaseModel):
```

Comment:
if we move the following validation(roughly) to OutputS3Storage. then we can keep this class inheriting from S3Config? I feel this would be cleaner than redefining the fields.
```
if not field.endswith("/"):
    raise ValueError('key_prefix must end with "/"')
return field
```